server/utils/userUpdatedSiteList.js

```diff
@@ -0,0 +1,8 @@
+const DISALLOWED_STUDIES = ['combined', 'files']
+const allowedStudies = (study) => !DISALLOWED_STUDIES.includes(study)
+
+export const recentStudyList = (studyList) =>
```

Comment:
what makes this `recent`? I think we can find a better variable name here.

Reply:
what do you think about userUpdatedStudyList as the name

Reply:
I don't think that really describes what we're doing. We're passing a function a list of studies that they have access to and then building a default target value map. What do you think of something like `defaultTargetValueMap`?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/174/commits/9a9f6bdc167cc457dc4e58047fb2414ce16025d2